3.12/README.md

```diff
@@ -0,0 +1,144 @@
+# What will 3.12 look like?
```

Comment:
This feels a bit more like a rough-ish, medium-term roadmap than a prediction of what users should expect in 3.12. Maybe:

```suggestion
# What lies beyond 3.11?
```

...or something.